CONTRIBUTING.md

```diff
@@ -62,3 +62,27 @@ We use [SassDoc] to document Bourbon. [Annotations] should be ordered:
 
 [SassDoc]: http://sassdoc.com
 [Annotations]: http://sassdoc.com/annotations
+
+## Goverance
+
+We, the maintainers of Bourbon, would like you to know what to expect when
+interacting with this project's repo.
+
+As a user of Bourbon, you can expect care to be put into official releases. The
+maintainers cannot guarantee that the `master` branch will have the same API
+forever.
+
+As someone who is reviewing pull requests or commenting on issues, you are
+expected to abide by our [code of conduct].
```

Comment:
How do you feel about `If you review pull requests or comment on issues, you are expected…`

`As someone` keeps with the `As role X` style, but "someone" is vague and makes this line harder to read